Bastien Durand/week_01/TheWordGuesser/TheWordGuesser.js

```diff
@@ -0,0 +1,84 @@
+// # Homework: The Word Guesser
+//
+// You'll create a simple word guessing game where the user gets infinite tries to guess the word (like Hangman without the hangman, or like Wheel of Fortune without the wheel and fortune).
+//
+// - Create two global arrays: one to hold the letters of the word (e.g. 'F', 'O', 'X'), and one to hold the current guessed letters (e.g. it would start with '\_', '\_', '\_' and end with 'F', 'O', 'X').
+// - Write a function called guessLetter that will:
+// - Take one argument, the guessed letter.
+// - Iterate through the word letters and see if the guessed letter is in there.
+// - If the guessed letter matches a word letter, changed the guessed letters array to reflect that.
+// - When it's done iterating, it should log the current guessed letters ('F__')
+// and congratulate the user if they found a new letter.
+// - It should also figure out if there are any more letters that need to be guessed,
+// and if not, it should congratulate the user for winning the game.
+// - Pretend you don't know the word, and call guessLetter multiple times with various letters to check that your program works.
+
+console.log('To play this game, simply write within the console guessedLetter(" "), Enter your letter within the quotation marks to see a result')
+
+const currentWordLetters = ['M', 'O', 'N', 'K', 'E', 'Y'];
+const currentGuesses = ['_', '_', '_', '_', '_', '_'];
+
+
+const guessedLetter = function (arg) {
+
+
+  if (arg === 'M') {
+    currentGuesses.splice(0, 1, 'M')
+
+  } else if (arg === 'O') {
+    currentGuesses.splice(1, 1, 'O')
+    console.log(`${arg} is Correct`)
+
+  } else if (arg === 'N') {
+    currentGuesses.splice(2, 1, 'N')
+    console.log(`${arg} is Correct`)
+
+  } else if (arg === 'K') {
+    currentGuesses.splice(3, 1, 'K')
+    console.log(`${arg} is Correct`)
+
+  } else if (arg === 'E') {
+    currentGuesses.splice(4, 1, 'E')
+    console.log(`${arg} is Correct`)
+
+  } else if (arg === 'Y') {
+    currentGuesses.splice(5, 1, 'Y')
+    console.log(`${arg} is Correct`)
+
+  } else if (arg !== 'M' || arg !== 'O' || arg !== 'N' || arg !== 'K' || arg !== 'E' || arg !== 'Y' ) {
+    console.log(`${arg} is Incorrect`);
+  }
```

Comment:
I can totally see what you were thinking here - and right idea! However your final else if statement could just be: 
```
} else {
  console.log(`${arg} is Incorrect`);
}
```
This is because you've already checked for each letter above so the only letters that will "make" it to the else statement will be those letters that aren't M, O, N, K, E, Y. 
(Let me know if that makes sense!)